### PR TITLE
Add group screen empty state

### PR DESCRIPTION
**Pull request by Ngkaokis**
ref #3903

### CODE REVIEW

**Comment by tung2744** [comment]
@Ngkaokis It seems this pr contains some changes in other prs and need a rebase

File: portal/src/graphql/adminapi/GroupsScreen.tsx [hunk 1]
```diff
@@ -1,7 +1,165 @@
...
+        }}
+      />
+    );
+  };
+
+interface GroupsScreenEmptyStateProps {
+  className?: string;
+}
+
+const GroupsScreenEmptyState: React.VFC<GroupsScreenEmptyStateProps> =
```
Comment (tung2744) [inline portal/src/graphql/adminapi/GroupsScreen.tsx:51]: Please use RolesAndGroupsEmptyView

File: portal/src/graphql/adminapi/GroupsScreen.module.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.emptyStateIcon {
+  @apply h-16 w-16;
+}
+
+.emptyStateTitle {
```
Comment (tung2744) [inline portal/src/graphql/adminapi/GroupsScreen.module.css:12]: Please remove the unused styles as well

**Comment by tung2744** [comment]
It seems this pr is still having conflicts

**Comment by Ngkaokis** [comment]
@tung2744 Rebased, thx :adore2x:

**Comment by Ngkaokis** [comment]
@tung2744 Rebased, thx :adore2x:

**Comment by Ngkaokis** [comment]
@tung2744 Rebased, thx :adore2x:

**Pull request merged**